src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -0,0 +1,107 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.AbsoluteEncoder;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkPIDController;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+public class Arm extends SubsystemBase {
+    public final CANSparkMax sprocketLeftMotor;
+    public final CANSparkMax sprocketRightMotor;
+    private final SparkPIDController sprocketLeftPIDController;
+    private final SparkPIDController sprocketRightPIDController;
+    private final RelativeEncoder sprocketLeftEncoder;
+    private final RelativeEncoder sprocketRightEncoder;     //Initialize
+    private final DutyCycleEncoder sprocketAbsoluteEncoder;
+    ArmFeedforward sprocketFeedforward = new ArmFeedforward(0, 0.0275, 0);
+    PIDController pidController = new PIDController(Constants.armConstants.controllerGain, 0, 0);
+    
+    public Arm() {
+        sprocketLeftMotor = new CANSparkMax(6, CANSparkMax.MotorType.kBrushless);
+        sprocketRightMotor = new CANSparkMax(7, CANSparkMax.MotorType.kBrushless);
+        sprocketRightMotor.setInverted(true);  //Inverts the motor because one motor is on the other side
```

Comment:
Put inversion status into Constants.Java

Reply:
I appreciate that the comment explained _why_ rather than what